Makefile

```
@@ -205,7 +198,7 @@ openapi-doc:	docs/devel_doc/openapi.json scripts/fix_openapi_doc.py	## Generate
 	python3 scripts/fix_openapi_doc.py < output.md > openapi2.md
 	rm output.md
 
-generate-documentation:	devel-doc schema	## Generate or regenerated content of the whole /docs subdirectory
+generate-documentation:	devel-doc schema docs/models	## Generate or regenerated content of the whole /docs subdirectory
```

checkmake (0.3.2)

[warning] Target "generate-documentation" should be declared PHONY.

(phonydeclared)